Buildings/Experimental/DHC/Networks/Steam/BaseClasses/PartialConnectionTwoPipe.mo

```diff
@@ -0,0 +1,283 @@
+within Buildings.Experimental.DHC.Networks.Steam.BaseClasses;
```

Comment:
Regarding these baseclasses, what I would suggest, instead of removing the original baseclasses that Antoine developed (which would require a conversion script in case it's being used elsewhere) would be to do the following:

1. Rename your baseclasses to `PartialConnectionTwoPipeTwoMedium`, `PartialDistributionTwoMedium` and `PartialDistributionTwoPipeTwoMedium`.
2. Move those baseclasses to `Buildings.Experimental.DHC.Networks.BaseClasses` (alongside Antoine's baseclasses)
3. Refactor Antoine's subclasses to extend yours, while redefining a common Medium. Something like:

```
partial model PartialConnection2Pipe
    extends Buildings.Experimental.DHC.Networks.BaseClasses.PartialConnectionTwoPipeTwoMedium(
        redeclare final package MediumSup=Medium,
        redeclare final package MediumRet=Medium)
    replaceable package Medium=Modelica.Media.Interfaces.PartialMedium
    "Medium model";
```

This alleviate the need for potentially complex conversion script (I actually don't know if you could add a new redeclare statement in every instance of the original `PartialConnection2Pipe`).

Please make sure that the signature (all of the parameters that are currently declared in the class) remains unchanged, and if not, either have extra parameters take final value, and missing parameters be declared after the extension.

I've filed a [ticket](https://github.com/lbl-srg/modelica-buildings/issues/2939) to take care of renaming Antoine's subclasses with spelled out number, so you can keep them the way they are now for the moment.

Reply:
@khinkelman After checking with Michael, we think it's actually clearer to use the number in the class names (for instance `PartialDistribution2Pipe` instead of `PartialDistributionTwoPipe`). This makes the names shorter and also makes the alphabetical order more consistent (2 < 3 but Two > Three).

So, when you rename the classes, please adopt that nomenclature `PartialConnection2Pipe2Medium` (etc). Thanks

Reply:
Revised as suggested.